controller/backing_image_controller.go

```diff
@@ -401,6 +401,20 @@ func (bic *BackingImageController) handleBackingImageDataSource(bi *longhorn.Bac
 			if err != nil {
 				return err
 			}
+
+			// For cloning, we choose the same node and disk as the source backing image
```

Comment:
nit: `handleBackingImageDataSource` is too longhorn to read easily. Suggest refactoring the codes in the future.